lib/pages/chat_list/chat_list.dart

```diff
@@ -193,10 +196,29 @@ class ChatListController extends State<ChatList>
 
   bool isGroupWithOnlyBotAndUser(Room room) {
     final client = Matrix.of(context).client;
+
+    // Check that participants are fully charged
```

Comment:
The next few lines seem bad for performance, especially during initial sync. Are they really necessary ? If its just to fix a few edge cases I'd rather them be removed.